src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/Form.vue

```diff
@@ -40,13 +41,19 @@ export default defineComponent({
     updatedFormData: (_formData: Record<string, any>) => true,
     updatedValidation: (_validation: boolean | null) => true,
   },
+  mounted() {
+    setTimeout(this.resetFormValidation, 100);
```

Comment:
Can you explain why this is needed? Would really like to avoid a timeout like this

Reply:
removed set timeout.